config/initializers/business_time.rb

```diff
@@ -0,0 +1,28 @@
+  require 'holiday_calendar'
+
+  module BusinessTimeHelpers
+    extend self
+
+    def holidays
+      scope
+        .to_a
+        .reduce([]) { |hols, y| hols << calendar.list_for_year(y) }
+        .flatten
+        .map { |h| Date.parse(h) } 
+    end
```

Comment:
Nice. But I think we need to find a way to make this a bit more obvious as to what it is doing.  It took me a console and a few minutest to work it out.